### DISCUSSION TITLE
having trouble finding the download link for the Beta update

### DISCUSSION THREAD

**Discussion by Keytarpaul** [nhielost/obs-midi-mg#93] [Q&A] [answered]
Could someone paste the link here or give the EXACT TEXT for the Beta Download update - Thank you
I was logged into github already

**Comment by nhielost** [comment, marked as answer]
Here is the link you are looking for.

[obs-midi-mg 3.0.0 Beta](https://github.com/nhielost/obs-midi-mg/releases/tag/3.0.0-beta)
